packages/pulpcore-wrappers/empty-file

```diff
@@ -0,0 +1 @@
+# really
```

Comment:
What's this fella for?

Reply:
I saw @evgeni complain about a missing Source0. I guess this is the hack to allow it to build. If it's done as part of python3-pulpcore, the problem goes away because that has a source.

It would also solve https://pulp.plan.io/issues/7663 if the installers are updated to use the libexec path.